unstructured/ingest/cli/cmds/kafka.py

```diff
@@ -0,0 +1,106 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.base.src import BaseSrcCmd
+from unstructured.ingest.cli.interfaces import CliConfig
+from unstructured.ingest.connector.kafka import KafkaWriteConfig, SimpleKafkaConfig
+
+CMD_NAME = "kafka"
+
+
+@dataclass
+class KafkaCliConfig(SimpleKafkaConfig, CliConfig):
+    @staticmethod
+    def get_cli_options() -> t.List[click.Option]:
+        options = [
+            click.Option(
+                ["--bootstrap-server"], required=True, type=str, help="Broker server hostname"
+            ),
+            click.Option(
+                ["--port"],
```

Comment:
@rbiseck3 How careful do we have to be with options that can be in multiple connectors? I know sometimes we have to prefix them to prevent a source connector from using the option of a destination connector that is named the same.

Reply:
It’ll raise an error when creating the CLI if there are any conflicting names. For now, this should be fine.